presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -192,6 +204,36 @@ public Duration getElapsedTime()
     }
 
     @JsonProperty
+    public Duration getExecutionTime()
+    {
+        return executionTime.convertToMostSuccinctTimeUnit();
+    }
+
+    @JsonProperty
+    public Duration getCpuTime()
+    {
+        return cpuTime;
+    }
+
+    @JsonProperty
+    public long getExecutionTimeMillis()
```

Comment:
Sorry I confused this with another method in a an earlier version of the PR. I'll get rid of the millis code.